book/_toc.yml

```
@@ -23,13 +23,20 @@ parts:
       - file: preliminary/checklist/earthdata
       - file: preliminary/checklist/earthengine
     - file: preliminary/skills_refresher_index
-      sections:
+      sections: 
         - file: preliminary/skills-refresher/swc
 - caption: Tutorials
   chapters:
     - file: tutorials/index
       sections:
         - file: tutorials/example/tutorial-notebook
+        - file: tutorials/data-access-and-format/index
+          sections:
+            - file: tutorials/data-access-and-format/overview
+            - file: tutorials/data-access-and-format/earthdata-search
```

```suggestion
            - file: tutorials/data-access-and-format/earthdata_search
```
Note failing CI check b/c of path typo

---

Also more typos to fix (not necessary to merge, but appreciated!) https://github.com/ICESAT-2HackWeek/ICESat-2-Hackweek-2023/actions/runs/5765054316/job/15630260619?pr=65